scp.c

```diff
@@ -2074,6 +2083,12 @@ sink(int argc, char **argv, const char *src)
 		omode = mode;
 		mode |= S_IWUSR;
 #ifdef WINDOWS
+		if (add_mark_of_web(np) == -1) {
+			if (verbose_mode) {
+				note_err("%s: add_mark_of_web failed\n", np);
```

Comment:
why is this message handled differently than in sftp-client (where is uses `debug` instead of `note_err`)?